elements/pfe-accordion/src/pfe-accordion-panel.html

```diff
@@ -1,4 +1,5 @@
 <div tabindex="-1">
+  <div class="pf-c-accordion__accent-mark"></div>
```

Comment:
Moving the accent to a separate element prevents the CSS variable from cascading to nested accordions inside the slot.